packs/data/feats.db/battle-cry.json

```diff
@@ -37,6 +37,17 @@
             },
             {
                 "key": "Note",
+                "outcome": [
+                    "criticalSuccess"
+                ],
+                "predicate": [
+                    {
+                        "eq": [
+                            "skill:itm:rank",
+                            4
+                        ]
+                    }
+                ],
```

Comment:
This is a prerequisite of having the feat. If someone adds it despite not meeting the prerequisite, we shouldn't take further measures to knee-cap its functionality.